src/Workspaces/CSharpTest/EmbeddedLanguages/RegularExpressions/CSharpRegexParserTests.cs

```diff
@@ -40,20 +40,19 @@ private SyntaxToken GetStringToken(string text)
             bool runSubTreeTests = true, [CallerMemberName]string name = "",
             bool allowIndexOutOfRange = false,
             bool allowNullReference = false,
-            bool allowOutOfMemory = false)
+            bool allowOutOfMemory = false,
+            bool allowDiagnosticsMismatch = false)
```

Comment:
needed as roslyn now tracks the latest corefx parsing.  However, what roslyn actually references in test code is the buggy .net parsing.   So we have to pass a flag saying that it's ok that our behaviors deviate from each other.